### PR TITLE
fix: tabs inkbar border lost

### LINKED ISSUE #24616: Tabs inkbar and border lost
https://ant.design/components/tabs/#components-tabs-demo-custom-tab-bar

image

### PR DESCRIPTION
**Pull request by xrkffgg**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
fix #24616

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix Tabs inkbar border lost when use with react-sticky.            |
| 🇨🇳 Chinese | 修复 Tabs 配合 react-sticky 使用时下划线遗失问题。          |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/tabs/style/card.less [hunk 1]
```diff
@@ -25,15 +25,15 @@
   // ========================== Top & Bottom ==========================
   &.@{tab-prefix-cls}-top,
   &.@{tab-prefix-cls}-bottom {
-    > .@{tab-prefix-cls}-nav {
+    .@{tab-prefix-cls}-nav {
```
Comment (afc163) [inline components/tabs/style/card.less:28]: 这个应该在 tabs 嵌 tabs 的时候有用。

**Comment by zombieJ** [comment]
跑一下这个例子：
截屏2020-06-01 下午6 07 13

**Comment by zombieJ** [comment]
得搞一个 reset 默认重置所有相关的，再附回去……
换成 `css in js` 就简单多了，可惜现在不是。

**Comment by xrkffgg** [comment]
Demo 点了点，没问题，不知道这种写法行不行

**Comment by xrkffgg** [comment]
@zombieJ 这种不行的话，你来改吧。

**Pull request merged**